File: cms_server/cms_server/Entities/HospResidentEntity.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace cms_server.Entities
{
    public class HospResidentEntity
    {
        public int res_pk { get; set; }
        public int user_pk { get; set; }
        public string doc_id { get; set; }
        public int spclty_pk { get; set; }
        public string pic_dest { get; set; }
        public string prefix { get; set; }
        public string first_name { get; set; }
        public string middle_name { get; set; }
        public string last_name { get; set; }
        public string suffix { get; set; }
        public string gender { get; set; }
        public string bio { get; set; }
        public string email { get; set; }
        public string mob_no { get; set; }
        public string is_active { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }

        //
        public string specialty { get; set; }
        public IFormFile img_attach { get; set; }
    }
}

File: cms_web/src/Services/Entities/HospResidentEntity.ts
interface HospResidentEntity {
  res_pk?: number;
  user_pk?: number;
  license_no?: string;
  dept_pk?: string;
  spclty_pk?: string;
  pic_dest?: string;
  doc_title?: string;
  prefix?: string;
  first_name?: string;
  middle_name?: string;
  last_name?: string;
  suffix?: string;
  gender?: "m" | "f";
  bio?: string;
  email?: string;
  mob_no?: string;
  is_active?: "y" | "n";
  encoded_at?: Date;
  encoder_pk?: string;
  esignature?: string;
  //ext
  res_name?: string;
  dept_name?: string;
  img_attach?: any;
  specialty?: string;
  total_consult?: string;
}

export interface HospResidentTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<HospResidentEntity>;
}

export default HospResidentEntity;
